recipes/pyrser/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyrser" %}
+{% set version = "2.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/LionelAuroux/{{ name }}/archive/master.tar.gz
+  sha256: 52244be9bb2727c5ba08ce311fbc4d6c8bbe801a3b07fe46d2c85e79cc372064
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - treematching
```

Comment:
How is this a dependency if it's not on PyPI?

Reply:
That was a mistake, removed!